### PR TITLE
Add info about deployment to ACA from Docker

### PR DESCRIPTION
**Pull request by bwateratmsft**
/cc @ghogen @gregvanl. Adding some docs for the new [Deploy to ACA feature](https://github.com/microsoft/vscode-docker/issues/3535) in the Docker extension. Whereas the "Deploy to App Service" feature does _not_ require the AppSvc extension, the "Deploy to ACA" feature _does_ require the ACA extension. If it is not present, you'll be prompted to install it, and taken to the extension install page automatically if you accept. I tried my best to phrase the prose talking about that to make the distinction clear.

Please hold off on merging until our 1.24.0 release, tentatively planned for 22 Feb.

### CODE REVIEW

File: docs/containers/quickstart-aspnet-core.md [hunk 1]
```diff
@@ -58,12 +58,12 @@ In this guide you will learn how to:
      Determining projects to restore...
      All projects are up-to-date for restore.
      net -> C:\source\repos\net\bin\Debug\net7.0\net.dll
-   
+
    Build succeeded.
        0 Warning(s)
        0 Error(s)
-
```
Comment (bwateratmsft) [inline docs/containers/quickstart-aspnet-core.md:65]: The editor *really* wanted to do these couple lines of whitespace changes, I literally couldn't stop it from doing them when editing in the browser...may as well just let it through

File: docs/containers/app-service.md [hunk 2]
```diff
@@ -64,15 +64,15 @@ Before deploying the image to an App Service, the image must be uploaded to a co
...
-## Deploy the image to Azure App Service
+## Deploy the image to Azure App Service or Azure Container Apps
 
-In the previous section, the image is pushed to a remote container registry. Now deploy this image to Azure App Service.
+In the previous section, the image is pushed to a remote container registry. Now deploy this image to Azure App Service or Azure Container Apps.
 
-1. In Docker Explorer, navigate to your image under Registries, right-click on the tag, and select **Deploy Image To Azure App Service...**.
+1. In Docker Explorer, navigate to your image under Registries, right-click on the tag, and select **Deploy Image To Azure App Service...** or **Deploy Image to Azure Container Apps...**.
 
     ![Deploy to Azure App Service](images/app-service/explorer-deploy-to-app-service.png)
```
Comment (bwateratmsft) [inline docs/containers/app-service.md:73]: Should the image be updated?

File: docs/containers/quickstart-container-registries.md [hunk 3]
```diff
@@ -94,7 +94,8 @@ For each tagged image in a repository, here are the actions that can be performe
 - **Pull Image**: copies the latest version of the image locally
```
Comment (bwateratmsft) [inline docs/containers/quickstart-container-registries.md:94]: Should the image on line 92 be updated?

File: docs/containers/quickstart-python.md [hunk 4]
```diff
@@ -178,11 +178,11 @@ You can use the command **Azure Container Registry: Build Image in Azure** to bu
...
+Once the container image is built, it should appear in the Container Registry with the tag you specified. Now that it's built, you can deploy it to Azure App Service or Azure Container Apps. The [Azure App Service](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azureappservice) extension is recommended for deployments to Azure App Service, and the [Azure Container Apps](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurecontainerapps) extension is required for deployments to Azure Container Apps. You can obtain both if you install the [Azure Tools extension pack](https://marketplace.visualstudio.com/items?itemName=ms-vscode.vscode-node-azure-pack), which includes a package of tools for a wide range of Azure development scenarios.
 
-1. Right-click on the image tag and choose **Deploy Image to to Azure App Service**.
+1. Right-click on the image tag and choose **Deploy Image to Azure App Service** or **Deploy Image to Azure Container Apps**.
 
    ![Deploy image to Azure App Service](images/app-service/deploy-image-to-azure-app-service.png)
```
Comment (bwateratmsft) [inline docs/containers/quickstart-python.md:187]: Should the image be updated?

File: docs/containers/quickstart-container-registries.md (hunk 3 above)
Comment (ghogen) [reply docs/containers/quickstart-container-registries.md:94]: Yes this should be updated to include the new item about ACA

File: docs/containers/quickstart-python.md (hunk 4 above)
Comment (ghogen) [reply docs/containers/quickstart-python.md:187]: Yes, this image should also be updated.

**Review by ghogen — APPROVED** [review]

**Comment by bwateratmsft** [comment]
These changes can be merged as soon as they are ready; we have done our 1.24.0 release.

**Pull request merged**